common.yml

```diff
@@ -1951,10 +1914,10 @@ buildvariants:
   - name: sign
     run_on: amazon1-2018-test
 
-- name: rhel82-s390x
+- name: rhel80-s390x
```

Comment:
@huan-Mongo same question here: was there a reason not to use the existing rhel80 distro? Just want to make sure this isn't undoing a decision you made intentionally.

Reply:
Here I use 8.2 to keep it consistent with the above change.

Reply:
The problem here is that there is no rhel82 distro for s390x (hence why the tasks weren't getting scheduled in the patch or on master). The RHEL8 distro that we do have for s390x is rhel80-zseries

Reply:
got it.